edgelet/edgelet-config/test/windows/sample_settings.tg.yaml

```diff
@@ -2,7 +2,7 @@
 # Configures the provisioning mode
 provisioning:
   source: "manual"
-  device_connection_string: "HostName=something.something.com;DeviceId=something;SharedAccessKey=something"
+  device_connection_string: "HostName=something.something.com;DeviceId=something;SharedAccessKey=QXp1cmUgSW9UIEVkZ2U="
```

Comment:
Just a heads up that we may get dinged in the SDL build for this string that looks like a secret.